terraform/environment/region/modules/schedule_runner/lambda.tf

```
@@ -28,7 +28,7 @@ module "schedule_runner" {
 }
 
 resource "aws_scheduler_schedule" "schedule_runner_hourly" {
-  name                = "schedule-runner-hourly"
+  name                = "schedule-runner-hourly-${data.aws_default_tags.current.tags.environment-name}-${data.aws_region.current.name}"
```

I don't think we need region as part of the name 

```suggestion
  name                = "schedule-runner-hourly-${data.aws_default_tags.current.tags.environment-name}"
```